src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -1,5 +1,6 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.collection.Stackable;
 import org.junit.Assert;
 import org.junit.Test;
 
```

Comment:
About testIsFull: If STACK_CAPACITY is 12 then 12 elements should be added to the stack and it should be FULL. It does not make sense. Incorrect fix.